tomviz/pybind11/ctvlib.cxx

```diff
@@ -0,0 +1,314 @@
+
+//
+//  ctvlib.cpp
+//
+//
+//  Created by Hovden Group on 5/6/19.
```

Comment:
This cannot be merged until this line is removed, and the standard C++ copyright header is inserted,
`/* This source file is part of the Tomviz project, https://tomviz.org/.
   It is released under the 3-Clause BSD License, see "LICENSE". */`
All C++ is BSD licensed, attribution is through Git history, and we cannot have lines with "All Rights Reserved" in them as they are not open source.

Reply:
@cryos my apologies! This is an easy fix, I'll remove these lines. It's a default licensing that's usually generated from producing projects on Xcode (heh, silly me)